acrg/name/name.py

```diff
@@ -1651,31 +1656,25 @@ def noon(dataset, site,keep_missing=False):
             return dataset[dict(time = ti)] 
         
             
-    def local_influence(dataset,site, keep_missing=False):
+    def local_influence(dataset,site,keep_missing=False):
         """
         Subset for times when local influence is below threshold.       
         Local influence expressed as a fraction of the sum of entire footprint domain.
-        """        
-        if not dataset.filter_by_attrs(standard_name="local_ratio"):
-            lr = local_ratio(dataset)
-        else:
-            lr = dataset.local_ratio
-            
-        pc = 0.1
+        """
+        pc = 0.1      #localness 'ratio' limit
+        lr = dataset.local_ratio
```

Comment:
Just to check - is this assuming `local_ratio` is a data variable on the input dataset already?